### PR TITLE
Refactoring Template.php 

### PR DESCRIPTION
**Pull request by annalinneajohansson**
Abstract: Enable flexibility when choosing a controller + view for purpose templates while keeping the same functionality as before.

### CODE REVIEW

**Comment by helsingborgsstad** [comment]
Task linked: [CU-865bkb62x Enabler: Refaktorera Template.php](https://app.clickup.com/t/865bkb62x)

**Comment by annalinneajohansson** [comment]
@annalinneajohansson Just a test!

File: library/Template.php [hunk 1]
```diff
@@ -116,23 +116,16 @@ public function sanitizeViewName($view)
...
-            \Municipio\Helper\Template::getViewPaths(),
-            "",
-            $view
-        ); // Remove view path
-
-        //Remove suffix
-        $view = trim(str_replace(".blade.php", "", $view), "/");
-
-        return str_replace("/", ".", $view);
+        return str_replace("/", ".", basename($view, '.blade.php'));
```
Comment (annalinneajohansson) [inline library/Template.php]: Replacing / shouldn't even be necessary? They're directories and removed with `basename()`.

**Comment by annalinneajohansson** [comment]
Branching off this into https://github.com/helsingborg-stad/Municipio/pull/445 to continue work with implementing purposes without snowballing this PR @sebastianthulin @nRamstedt

**Pull request closed without merging**